app/models/miq_policy_set.rb

```diff
@@ -116,6 +118,10 @@ def self.import_from_yaml(fd)
   end
 
   def self.seed
+    seed_model(
+      self, :has_many => [:miq_policies, -> (policy_set, policy) { policy_set.add_member(policy) }]
```

Comment:
@moolitayer First of all, this is awesome!  I'd like to see tests for this too... this api isn't immediately obvious to me, but I don't have useful suggestions so I'm fine with it... but maybe we can refine this api after we have some tests.